src/posts/keymaps/Wimads.md

```diff
@@ -12,13 +12,13 @@ isTapDanceEnabled: false
 keybindings: [Graphics/CAD]
 keyboard: 3W6
 keyCount: 36
-keymapImage: https://github.com/Wimads/qmk_firmware/blob/master/keyboards/keycapsss/3w6_2040/keymaps/Wimads/keymapdb/Layers.png
+keymapImage: https://raw.githubusercontent.com/Wimads/qmk_firmware/master/keyboards/keycapsss/3w6_2040/keymaps/Wimads/keymapdb/Layers.png
 keymapUrl: https://github.com/Wimads/qmk_firmware/blob/master/keyboards/keycapsss/3w6_2040/keymaps/Wimads
 languages: [English, Dutch]
-layerCount: 3
+layerCount: 6
```

Comment:
I know my actual keymap.c file has 6 layers, but that's mostly noise. In practical use its effectively 3 layers the way I've shown it in the images. Wouldn't the practical count be more relevant when filtering through the database, as opposed to the technical count in the code?

Reply:
For the sake of consistency and simplicity, I figured that relying on an objective metric like the number of non-empty layers actually defined in the firmware is better than introducing a discrepancy between practical count and technical count. Furthermore, most of the keymaps at this point were added by me, and I don't want to have to guess how many layers does that person actually use in day-to-day life.